app/boards/arm/corneish_zen/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
```

Comment:
Oh! Old checkouts might still have this, but this got deprecated in the Zephyr 3.0 upgrade. See https://zmk.dev/blog/2022/04/02/zephyr-3-0#uf2-builds for how to do this the right way now.

Reply:
Yes, that was it. I had the new parts in the config, but didn't remove the old parts. building fine now